controllers/applications.js

```diff
@@ -1,44 +1,39 @@
-const fs = require('fs');
-const Applications = require('../data/applications.json');
+const Applications = require('../models/Applications');
+
+const listApplication = (req, res) => {
+  Applications.find(req.query)
+    .then((application) => res.status(200).json(application))
+    .catch((error) => res.status(404).json({ message: error }));
```

Comment:
status code 400

Reply:
Sorry, I thought it was a 404 status when, in this case, the id was not found